gateways/adyen/request_builder.go

```diff
@@ -130,3 +138,12 @@ func buildVoidRequest(voidRequest *sleet.VoidRequest, merchantAccount string) *p
 	}
 	return request
 }
+
+// NAifEmpty returns "NA" if the parameter is an empty string. Adyen best practices
+// for Level 2/3 data are to set the value of any fields that are unknown to "NA"
+func NAifEmpty(str string) string {
```

Comment:
can we make this a little more general? Like DefaultIfEmpty that takes a default value and returns that instead of NA? ANd we can move that to sleet common

Reply:
Done